hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/NodeDecommissionManager.java

```diff
@@ -527,7 +528,7 @@ public synchronized void startMaintenance(DatanodeDetails dn, int endInHours)
     long maintenanceEnd = 0;
     if (endInHours != 0) {
       maintenanceEnd =
-          (System.currentTimeMillis() / 1000L) + (endInHours * 60L * 60L);
+          (Time.monotonicNow() / 1000L) + (endInHours * 60L * 60L);
```

Comment:
This is related to `NodeStatus` mentioned in https://github.com/apache/ozone/pull/8096/#discussion_r1997654326, I think it should keep using `System.currentTimeMillis()`.